jobs/e2e-test-cypress.groovy

```diff
@@ -91,7 +94,7 @@ REVISION_DESCRIPTION = params.REVISION_DESCRIPTION ?: params.GIT_REVISION;
 currentBuild.displayName = ("${currentBuild.displayName} " +
                             "(${REVISION_DESCRIPTION})");
 
-// We use the build name as a unique identifier for user notifications. 
+// We use the build name as a unique identifier for user notifications.
```

Comment:
**note:** a lot of changes in this file (and in `e2e-test`) are formatting changes (adding newlines, removing whitespaces, etc).